src/handler/grpc/request/event.rs

```diff
@@ -52,27 +73,41 @@ impl Event for Eventer {
             for item in put_items.iter() {
                 // cache parquet
                 if cfg.cache_latest_files.cache_parquet {
-                    if let Err(e) =
-                        infra::cache::file_data::download(TRACE_ID_FOR_CACHE_LATEST_FILE, &item.key)
-                            .await
-                    {
-                        log::error!("Failed to cache file data: {}", e);
-                    }
-                }
-                // cache index for the parquet
-                if cfg.cache_latest_files.cache_index && item.meta.index_size > 0 {
-                    if let Some(ttv_file) = convert_parquet_idx_file_name_to_tantivy_file(&item.key)
-                    {
+                    log::info!("get_files_from_notifier: {grpc_addr}");
+                    if let Err(e) = get_files_from_notifier(&grpc_addr, &item.key).await {
+                        if e.to_string() != *DOWNLOAD_FROM_NODE_NOT_ENABLED {
```

Comment:
Here we use error to check, maybe if it doesn't enable we no need to call the function.